libraries/octree/src/OctreeEntitiesFileParser.h

```diff
@@ -16,12 +16,14 @@
 
 #include <QByteArray>
 #include <QVariant>
+#include <QUrl>
```

Comment:
```suggestion
#include <QUrl>
#include <QVariant>
```
Best to order alphabetically if can.